.circleci/config.yml

```diff
@@ -56,138 +56,99 @@ aliases:
-
       - run:
           name: Setup Build
           command: |
             mkdir -p ~/test-reports
-            echo "2.3" > .ruby-version
-            brew untap caskroom/versions
+            echo ${_RUBY_VERSION} > .ruby-version
+            brew untap caskroom/versions || true
             brew install shellcheck
             bundle check --path .bundle || bundle install --jobs=4 --retry=3 --path .bundle
-
-      - *cache_save_homebrew
-      - *cache_save_bundler
-
-      - run:
-          name: Check PR Metadata
-          command: bundle exec danger || echo "danger failed"
-
-      - *cache_restore_rubocop
-
-      - run: bundle exec fastlane execute_tests
-
-      - *cache_save_rubocop
-
-      - store_test_results:
-          path: ~/test-reports
-      - store_artifacts:
-          path: ~/test-reports/rspec
-          destination: test-reports
-
-      - run:
-          name: Post Test Results to GitHub
-          command: bundle exec danger || echo "danger failed"
-          when: always  # Run this even when tests fail
-
-  "Execute tests on macOS (Xcode 11.0.0, Ruby 2.5)":
-    macos:
-      xcode: "11.0.0"
-    environment:
-      CIRCLE_TEST_REPORTS: ~/test-reports
-      LC_ALL: en_US.UTF-8
-      LANG: en_US.UTF-8
-    shell: /bin/bash --login -eo pipefail
-    steps:
-      - *cache_restore_git
-      - checkout
-      - *cache_save_git
-      - *cache_restore_bundler
-      - *cache_restore_homebrew
-
       - run:
-          name: Setup Build
+          name: debug | build environment
           command: |
-            mkdir -p ~/test-reports
-            echo "2.5" > .ruby-version
-            brew untap caskroom/versions
-            brew install shellcheck
-            bundle check --path .bundle || bundle install --jobs=4 --retry=3 --path .bundle
-
+            ruby -v
       - *cache_save_homebrew
       - *cache_save_bundler
-
       - run:
           name: Check PR Metadata
           command: bundle exec danger || echo "danger failed"
-
       - *cache_restore_rubocop
-
       - run: bundle exec fastlane execute_tests
-
       - *cache_save_rubocop
       - store_test_results:
           path: ~/test-reports
       - store_artifacts:
           path: ~/test-reports/rspec
           destination: test-reports
-
       - run:
           name: Post Test Results to GitHub
           command: bundle exec danger || echo "danger failed"
           when: always  # Run this even when tests fail
+  'Execute tests on macOS (Xcode 10.2.1, Ruby 2.4)':
+    <<: *tests_macos
+    macos:
+      xcode: '10.2.1'
+    environment:
+      _RUBY_VERSION: '2.4'
+  'Execute tests on macOS (Xcode 11.0.0, Ruby 2.5)':
+    <<: *tests_macos
+    macos:
+      xcode: '11.0.0'
+    environment:
+      _RUBY_VERSION: '2.5'
+  'Execute tests on macOS (Xcode 11.3.0, Ruby 2.6)':
+    <<: *tests_macos
+    macos:
+      xcode: '11.3.0'
+    environment:
+      _RUBY_VERSION: '2.6'
 
-  "Execute tests on Ubuntu":
+  'tests_ubuntu':
```

Comment:
Ah, just noticed: This should also be turned back into `Execute tests on Ubuntu`.

Reply:
done